### DISCUSSION TITLE
How i can update SKU in variations?

### DISCUSSION THREAD

**Discussion by Denys-Vasyliev** [etsy/open-api#928] [Q&A]
I have listings with 50+ variations (8 colors and 8 sizes each), every pair color-size have sku like "style-color-size" (example: 01red12)

how I can update all or few sku if i work with copy/draft?

I try getListingInventory and updateListingInventory, but this is dont work fo me

**Comment by showtroylove** [comment]
@Denys-Vasyliev regrettably it's difficult to diagnose your problem without additional information.  Can you provide the use case, input data, expected output data, errors encountered, and steps to reproduce?

**Reply by hemesh** [reply]
To expand a bit, the V2 endpoint doesn't allow an 'offering_id' when updating quantities unlike the V2 endpoint that did. How does it know which offering to update? Seems bonkers!

**Comment by showtroylove** [comment]
@Hemesh, you can use the SKU as the identifier.  Your SKU should be your
Primary Key, if you will, for each of your product variations.
